.github/workflows/learning-web-branch.yaml

```diff
@@ -28,7 +28,7 @@ jobs:
 
       - name: Install modules
         working-directory: learning/web
-        run: yarn install --frozen-lockfile
```

Comment:
If [`--frozen-lockfile`](https://classic.yarnpkg.com/en/docs/cli/install/#toc-yarn-install-frozen-lockfile) is throwing an error it's cause the lockfile is out of date so just running `yarn` to update `yarn.lock` should fix it